src/main/java/Duke.java

```diff
@@ -1,10 +1,154 @@
+        tasks[whichTask].setStatus(done);
+
+        if(whichTask > numTasks){
+            System.out.println(ERROR_OUT_OF_BOUND);
+        }
+
+        if(done){
+            System.out.println(MESSAGE_DONE);
+            System.out.println("\t" + tasks[whichTask].toString());
+        } else {
+            System.out.println(MESSAGE_NOT_DONE);
+            System.out.println("\t" + tasks[whichTask].toString());
+        }
+    }
+
+    public static void processUserInput(String userInput){
+        String[] splitInput = userInput.split(" ", 2);
+        String command = splitInput[0];
+        String parameters = "false input";
+        if (splitInput.length > 1){
+            parameters = splitInput[1];
+        }
+
+        switch (command){
+        case("list"):
+            listTasks();
+            break;
+        case("mark"):
+            //assume input of length 2
+            setTask(Integer.parseInt(splitInput[1]) - 1,true);
+            break;
+        case("unmark"):
+            setTask(Integer.parseInt(splitInput[1]) - 1, false);
+            break;
+        case("todo"):
+            addTodo(parameters);
+            break;
+        case("deadline"):
+            addDeadline(parameters);
+            break;
+        case("event"):
+            addEvent(parameters);
+            break;
+        default:
+            addTask(userInput);
+            break;
+        }
+    }
+    public static void greetUser(){
+        /*
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        */
+
+        final String LOGO = "\n" +
+        "     _________________________________________\n" +
+        "   /  _____________________________________   \\ \n" +
+        "   | |                                     |  | \n" +
+        "   | |  C:\\> Initiating programme_         |  | \n" +
+        "   | |                                     |  | \n" +
+        "   | |                                     |  | \n" +
+        "   | |                                     |  | \n" +
+        "   | |                                     |  | \n" +
+        "   | |                                     |  | \n" +
+        "   | |_____________________________________|  | \n" +
+        "    \\_____________________________________/ \n" +
+        "       \\________________________________/ \n" +
+        "        _________________________________ \n" +
+        "   _-'.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.--- `-_ \n" +
+        "_-'.-.-. .---.-.-.-.-.-.-.-.-.-.-.-.-.-.-.--..-.-.`-_ \n";
+
+        System.out.println(LOGO);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+    }
+
+    public static boolean toExit(String userInput) {
```

Comment:
can consider changing the method name to reflect the boolean nature of the method (toExit does not sound like it's a boolean method)